Library/ViewModels/FeatureFlagToolsViewModelTests.swift

```diff
@@ -7,7 +7,7 @@ import ReactiveSwift
 import XCTest
 
 final class FeatureFlagToolsViewModelTests: TestCase {
-  private let vm = FeatureFlagToolsViewModel()
+  private let vm: FeatureFlagToolsViewModelType = FeatureFlagToolsViewModel()
```

Comment:
This is so that we don't have direct access to `VM`'s properties but have to go through `input`/`output`

Reply:
Thanks for changing this! 👍🏾